packages/aws-rfdk/lib/deadline/test/repository.test.ts

```diff
@@ -1224,3 +1225,45 @@ test('imports repository settings', () => {
   const installerGroup = repository.node.tryFindChild('Installer') as AutoScalingGroup;
   expect(installerGroup.userData.render()).toContain(`aws s3 cp '${repositorySettings.s3ObjectUrl}'`);
 });
+
+test('secret manager enabled', () => {
```

Comment:
We should have a test case where we don't set the `enabled` prop or the `credentials` prop and we confirm that Secrets Management is enabled and a credentials Secret gets generated.